src/olympia/hero/tests/test_views.py

```diff
@@ -67,14 +76,53 @@ def test_basic(self):
         }
 
 
+class TestSecondaryHeroShelfViewSet(TestCase):
+    def setUp(self):
+        self.url = reverse_ns('hero-secondary-list', api_version='v5')
+
+    def test_basic(self):
+        response = self.client.get(self.url)
+        assert response.status_code == 200
+        assert response.json() == {'results': []}
+
+        hero_a = SecondaryHero.objects.create(
+            headline='Its a héadline!',
+            description='foo')
+        hero_b = SecondaryHero.objects.create(
+            headline='%^*',
+            description='',
+            cta_url='http://goo.gl',
+            cta_text='goozilla')
+        SecondaryHero.objects.create(
+            headline='dfdfd!',
+            description='dfdfd')
+
+        # The shelf isn't enabled so still won't show up
+        response = self.client.get(self.url)
+        assert response.json() == {'results': []}
+
+        hero_a.update(enabled=True)
+        hero_b.update(enabled=True)
+        # don't enable the 3rd PrimaryHero object
+        response = self.client.get(self.url)
+        assert response.status_code == 200
+        assert response.json() == {
+            'results': [
+                SecondaryHeroShelfSerializer(instance=hero_a).data,
+                SecondaryHeroShelfSerializer(instance=hero_b).data]}
```

Comment:
Either here or in the serializer; we should test that html-tags are exported the way we expect them to.

Reply:
Perhaps I'll just leave out all mention of escaping until the html (escaping) has been implemented (or it's been confirmed frontend doesn't want html anyway)